tests/test_views.py

```diff
@@ -170,16 +172,29 @@ def test_chat_response_default(
 
         test_data = [
             {'role': 'user', 'content': 'What is 2+2?'},
-            {'role': 'assistant', 'content': 'It is 4'}
+            {'role': 'assistant', 'content': 'It is 4'},
+            {'role': 'user', 'content': 'And what else?'},
```

Comment:
I've added a check that validates the last message from the list must be from a user. Otherwise its confusing to store the values. Also, I think we should be processing only one question at the time.